src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -4,8 +4,10 @@ import { RegistrationFormCreateBidderMutation } from "__generated__/Registration
 import { RegistrationFormCreateCreditCardMutation } from "__generated__/RegistrationFormCreateCreditCardMutation.graphql"
 import { CreditCardInput } from "Apps/Order/Components/CreditCardInput"
 import { ConditionsOfSaleCheckbox } from "Components/Auction/ConditionsOfSaleCheckbox"
-import { Form, Formik, FormikProps } from "formik"
-import React from "react"
+import { ErrorModal } from "Components/Modal/ErrorModal"
```

Comment:
We should update this `ErrorModal` to use the [artsy/palette](https://palette.artsy.net/elements/dialogs/modal/) modal under the hood.

Reply:
Shall we ticket this follow up? Implement it now? What are your thoughts on next steps here?

Reply:
I'm discussing this with the Design Systems group. I think we can circle back to this.